### DISCUSSION TITLE
How to block hackers trying to figure out my software stack

### DISCUSSION THREAD

**Discussion by juniorforlife** [vercel/community#6213] [Help] [answered]
### Question

Hello. My SvelteKit application, which is hosted on Vercel, has been experiencing a significant number of requests that look like the following:
<img width="1577" alt="Screenshot 2024-03-06 at 17 11 12" src="https://github.com/vercel/vercel/assets/68360917/210ccfbe-3d98-4588-a80e-4add42a89f33">

It appears as though someone is attempting to determine if my site is built on WordPress, potentially to exploit it.

Does Vercel automatically filter out these types of harmful requests? What steps can I take through the Vercel settings to better safeguard against such threats?

**Comment by leerob** [comment, marked as answer]
Hey! If you believe you are being attacked with malicious traffic, you could turn on Attack Challenge Mode temporarily.

https://vercel.com/changelog/prevent-malicious-traffic-with-attack-challenge-mode-for-vercel-firewall

**Reply by juniorforlife** [reply]
Thank you 🙏
